src/components/Dashboard/MainContainer/MainContainerUtils.js

```diff
@@ -13,3 +13,21 @@ export const getTaskListAsArray = (snapValue = {}) => {
 
   return taskList;
 };
+
+export const deleteDragOver = (event) => {
+  event.preventDefault();
+  document.getElementById('delete-zone').classList.add('drag-in');
+};
+
+export const deleteDragLeave = (event) => {
+  event.preventDefault();
+  document.getElementById('delete-zone').classList.remove('drag-in');
+};
```

Comment:
to avoid code duplication, you can combine `deleteDragOver` and `deleteDragLeave` into 1 toggle function